### DISCUSSION TITLE
Welcome to Office 365 / Azure Audit Log Collector discussions

### DISCUSSION THREAD

**Discussion by ddbnl** [ddbnl/office365-audit-log-collector#23] [Announcements]
##  Audit log collector discussions

This tool is being actively maintained, so feel free to ask any questions, make suggestions, or request new features. For concrete problems please open an issue.

**Comment by Emkadeu** [comment]
Thx for your work :)

**Reply by ddbnl** [reply]
No problem, hope the tool is useful to you!

**Reply by Emkadeu** [reply]
Yes sir!

**Comment by brummybird** [comment]
This is a great tool thank you!! I wondered if we are pulling AzureAD Risky Users from the 365 logs? Im using Graylog and dont seem to have the AADRiskyUsers table as a field. Am I missing something? Thanks again Lyndsey
